include/dlaf/util_math.h

```diff
@@ -106,6 +107,23 @@ constexpr ArithmeticT generic_integer_op(const TA a, const TB b, BinaryOp op) {
   return op(integral_cast<ArithmeticT>(a), integral_cast<ArithmeticT>(b));
 }
 
+/// Convert a percentage to an index.
+///
+/// @tparam ArithmeticT must be a floating point type
+/// @param size the size of the array
+/// @pre @p size >= 0,
+/// @param percent index corresponding to the first @percent percent of elements
+/// @pre @p percent >= 0 and @p percent <= 100,
+/// @return the end index corresponding to the first @percent percent of elements
+template <typename ArithmeticT>
```

Comment:
Purely aesthetic, but I think we mostly use `template <class ArithmeticT>`.

Reply:
I tried to follow the DLAF convention, but this slipped. Changed to `class`.